evap/staff/forms.py

```diff
@@ -682,16 +682,16 @@ class TextAnswerForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['original_answer'].disabled = "True"
+        self.initial['original_answer'] = self.instance.original_answer or self.instance.answer
 
     class Meta:
         model = TextAnswer
-        fields = ("original_answer", "reviewed_answer",)
+        fields = ("answer", "original_answer",)
 
-    def clean_reviewed_answer(self):
-        reviewed_answer = normalize_newlines(self.cleaned_data.get('reviewed_answer'))
-        if reviewed_answer == normalize_newlines(self.instance.original_answer) or reviewed_answer == '':
-            return None
-        return reviewed_answer
+    def save(self, *args, **kwargs):
+        if self.instance.original_answer == self.instance.answer:
+            self.instance.original_answer = None
```

Comment:
you're checking against the values of instance, which don't care about the values entered in the form as far as i know, are you sure this works?

how about putting this into `clean_original_answer`? similar to how this was done before. also, there was some stuff about normalize_newlines in there and a check for an empty string, make sure there's no unintended behavior change.